test/Experimental/JuLie/schur_polynomials.jl

```diff
@@ -0,0 +1,70 @@
+import .JuLie: schur_polynomial_cbf
```

Comment:
This is where the error occurs. I don't see how this can work for you locally? Do you perhaps mean
```suggestion
import Oscar.JuLie: schur_polynomial_cbf
```

Reply:
Alright. Honestly, I don't know what precisely either one of the two statements is doing. But let's try it out.